samples/dnn/yolo_object_detection.cpp

```diff
@@ -1,3 +1,23 @@
+//! Brief Sample of using OpenCV dnn module in real time with device capture, video and image.
+//! Questions and suggestions email to: Alessandro de Oliveira Faria cabelo@opensuse.org or blog http://assuntonerd.com.br
+/*
+ * VIDEO DEMO:
+ * https://www.youtube.com/watch?v=NHtRlndE2cg
+ *
+ * COMPILE:
```

Comment:
OpenCV builds samples by itself using `-DBUILD_EXAMPLES=ON` CMake's flag. It's not necessary I guess.

Reply:
Ok thanks

Reply:
removed